### PR TITLE
Create Mask.py

### PR DESCRIPTION
**Pull request by rushi-k12**
## Description
In this code, inputs['input_ids'] contains the tokenized input IDs, and attention_masks is a tensor with the same shape, where 1s indicate tokens and 0s indicate padding tokens. Ensure that you pass inputs['input_ids'] and attention_masks to your model during training.

### CODE REVIEW

**Comment by rbharath** [comment]
@rushi-k12 Can you come by and join one of our office hours https://forum.deepchem.io/t/announcing-the-deepchem-office-hours/293? We can't merge this PR as-is since it is a script rather than a library contribution, but I can discuss ways we can work this material into a tutorial at OH

**Comment by rushi-k12** [comment]
@rbharath I updated the file with a Google Colab file, please consider it now with the hactoberfest-accepted label

**Comment by rbharath** [comment]
@rushi-k12 We can't merge it in unless it's structured for contribution. Try to come by our OH and I would be happy to walk you through the process
